src/Apps/Checkout/Components/PaymentForm/index.tsx

```diff
@@ -0,0 +1,151 @@
+import { Elements, useElements, PaymentElement } from "@stripe/react-stripe-js"
```

Comment:
We should always avoid `index` files! Check out the [best practices doc](https://github.com/artsy/force/blob/main/docs/best_practices.md#should-i-use-an-index-file). 

Something like `Checkout/Components/PaymentForm.tsx` is fine (not everything needs its own folder, unless its particularly complex)